sdarq/backend/src/app.py

```diff
@@ -168,7 +168,9 @@ def submit():
         del json_data['Ticket_Description']
 
         product_id = dojo_helper.dojo_create_or_update(dojo_name, parse_json_data.prepare_dojo_input(json_data), product_type, user_email, appsec_slack_channel, security_champion, dojo_host_url)
-
+        if not product_id:
+            logging.error('DefectDojo did not respond correctly when attempting to update the product')
```

Comment:
if the raise message will be logged then you don't need logging.error here; if not then you need it at 233.